packages/mantine/src/theme/Theme.tsx

```diff
@@ -22,12 +23,12 @@ export const plasmaTheme: MantineThemeOverride = {
         fontFamily: 'canada-type-gibson, sans-serif',
         fontWeight: 500,
         sizes: {
-            h1: {fontSize: 48, lineHeight: '56px', fontWeight: 300},
-            h2: {fontSize: 32, lineHeight: '40px', fontWeight: 500},
-            h3: {fontSize: 24, lineHeight: '32px', fontWeight: 500},
-            h4: {fontSize: 18, lineHeight: '28px', fontWeight: 300},
-            h5: {fontSize: 14, lineHeight: '20px', fontWeight: 500},
-            h6: {fontSize: 12, lineHeight: '16px', fontWeight: 500},
+            h1: {fontSize: 48, lineHeight: undefined, fontWeight: 300},
+            h2: {fontSize: 32, lineHeight: undefined, fontWeight: 500},
+            h3: {fontSize: 24, lineHeight: undefined, fontWeight: 500},
+            h4: {fontSize: 18, lineHeight: undefined, fontWeight: 300},
+            h5: {fontSize: 14, lineHeight: undefined, fontWeight: 500},
+            h6: {fontSize: 12, lineHeight: undefined, fontWeight: 500},
```

Comment:
I think so, but I am not sure. I thought it would get overriden by [these](https://github.com/mantinedev/mantine/blob/master/src/mantine-styles/src/theme/default-theme.ts#L81-L87) if I didn't